clash_lib/Cargo.toml

```diff
@@ -36,6 +36,9 @@ network-interface = "1.1.3"
 base64 = "0.22"
 uuid = { version = "1.8.0", features = ["v4", "fast-rng", "macro-diagnostics", "serde"] }
 
+# boring = "4.5.0"
```

Comment:
remove comments plz and in the Cross.toml as well

Reply:
i want left some comments in `Cross.toml`. Just in case of i forget it has those features. like `pre-built` or `zig` or `passthrough`.